components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/JMSConstants.java

```diff
@@ -179,7 +179,9 @@
 
     public static final String HYPHEN_REPLACEMENT_STR = "_DASHED_";
 
-    public static final String DEFAULT_HYPHEN_SUPPORT = HYPHEN_MODE_NONE;     
+    public static final String DEFAULT_HYPHEN_SUPPORT = HYPHEN_MODE_NONE;
+
+    public static final String JMS_RETRY_DURATION = "transport.jms.retry.duration";
```

Comment:
We do not introduce new APIs in patch fixes. Not sure if this for a patch fix, but if this minor then it is fine.